src/bls381/mod.rs

```diff
@@ -31,16 +31,7 @@ pub mod utils;
 mod core;
 mod iso;
 
-#[cfg(target_pointer_width = "64")]
-mod iso_constants_x64;
-#[cfg(target_pointer_width = "64")]
-pub(crate) use iso_constants_x64::{
-    ISO11_XDEN, ISO11_XNUM, ISO11_YDEN, ISO11_YNUM, ISO3_XDEN, ISO3_XNUM, ISO3_YDEN, ISO3_YNUM,
-};
```

Comment:
Why remove the 64-bit versions? Don't we want to use them in native builds?

Reply:
I think substrate only supports 32bit wasm.

Reply:
Exactly, wasm runtime is built with 32bit and I met `SignatureVerificationFailed` when keeps these 64bit compilation options so remove here.